src/app/cluster-template/template.html

```diff
@@ -22,8 +22,10 @@
     <button fxLayoutAlign="center center"
             mat-flat-button
             type="button"
+            [matTooltip]="projectViewOnlyToolTip"
+            [matTooltipDisabled]="canCreate()"
             (click)="create()"
-            [disabled]="!canCreate()">
+            [disabled]="!canCreate() || !hasCurrentGroupConfig">
```

Comment:
has _currentGroupConfig is returned from API, at first the button isn't disabled.

Reply:
But what if API request is complete and there is still no `_currentGroupConfig` value?

Reply:
then it should stay disabled.

Reply:
Again, `MemberUtils.hasPermission` states otherwise:
```
 // Allow access if the access configuration is missing.
    if (!groupConfig || !groupConfig[viewName]) {
      return true;
    }

Reply:
I'm replacing this logic with a loading checker

Reply:
If we want to wait for API call to finish then please add separate property for that.